build.gradle

```diff
@@ -78,30 +80,33 @@ dependencies {
     }*/
 
     compile('org.springframework.boot:spring-boot-starter-data-jpa')
-    compile 'com.github.warchant:ed25519-sha3-java:1.2.0'
+    compile ("com.github.warchant:ed25519-sha3-java:${ed25519Version}")
```

Comment:
Shouldn't we use 'implementation' everywhere instead of 'compile'?

Reply:
It's true for Android applications, for plain apps we can use both versions of gradle commands